wpimath/src/main/native/include/frc/geometry/Quaternion.h

```diff
@@ -5,6 +5,7 @@
 #pragma once
 
 #include <wpi/SymbolExports.h>
+#include <wpi/json.h>
```

Comment:
Since the JSON serialization functions only need a reference to a wpi::json, put this at the top of headers that use serialization, after the #includes.
```cpp
namespace wpi {
class json;
}  // namespace wpi
```
Then put `#include <wpi/json.h>` in the corresponding .cpp file.